src/bundle/Resources/richtext/stylesheets/docbook/xhtml5/edit/core.xsl

```diff
@@ -334,6 +343,13 @@
           <xsl:value-of select="@ezxhtml:class"/>
         </xsl:attribute>
       </xsl:if>
+      <xsl:if test="@mark">
+        <xsl:attribute name="style">
+          <xsl:if test="@mark = 'disc'">list-style-type:disc;</xsl:if>
+          <xsl:if test="@mark = 'circle'">list-style-type:circle;</xsl:if>
+          <xsl:if test="@mark = 'square'">list-style-type:square;</xsl:if>
+        </xsl:attribute>
+      </xsl:if>
```

Comment:
I've given it more thought and did some investigation. There's no easy way to make this solution clean in XSLT 1.0. I don't see a reason why we should restrict these styles to the only one supported by OE. It's an artificial limitation and the maintenance cost of the possible solutions is too high.

Moreover validation here is not necessary. DocBook is the source of truth. So if DocBook has set mark, we can trust this value.

It can be made as simple as 
```suggestion
      <xsl:if test="@mark">
        <xsl:attribute name="style">
          <xsl:value-of select="concat('list-style-type:', @mark, ';')"/>
        </xsl:attribute>
      </xsl:if>
```

Reply:
I am not too fond of putting XHTML atttribute values directly into docbook unless we use `ezxhtml` namespace but that is just my option. It might be that we do that elsewhere anyway

Reply:
I wasn't aware of the `extractStyleValue` template  that removes the `list-style-type:` part... With that removed, I have no problem how the docbook looks like..
Fixed in f7d30bfbc903411207f2739be293b5af84fd4a8a